testing/wireguard/APKBUILD

```diff
@@ -0,0 +1,94 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+
+_flavor=${FLAVOR:-grsec}
```

Comment:
I think I should be creating 2 x `APKBUILD`:
* `wireguard-grsec`
* `wireguard-vanilla` 

(as is done for `zfs` & `spl`)